src/constants/index.js

```diff
@@ -1 +1,10 @@
-export const BaseUrlApiEngine = 'https://bot-builder-engine-2242d70bd3b3.herokuapp.com'
\ No newline at end of file
+//prod
+export const BaseUrlApiEngine = 'https://bot-builder-engine-2242d70bd3b3.herokuapp.com'
+
+// local
+// export const BaseUrlApiEngine = 'http://localhost:8080'
+export const  KEYCLOAK_CLIENT_CONFIG = {
+    url: `http://localhost:8080`,
```

Comment:
alterar para pegar variavel de ambiente de forma no segura, no react as envs var no client_side sao dessa forma
REACT_APP_${valor}

exemplo
REACT_APP_KEYCLOACK_URL

Reply:
Por estar usando vite vou ter que usar com VITE_${VALUE} https://vitejs.dev/guide/env-and-mode#env-files